python/ray/util/scheduling_strategies.py

```diff
@@ -72,9 +72,87 @@ def __init__(
         self._fail_on_unavailable = _fail_on_unavailable
 
 
+def validate_values(values):
+    for value in values:
+        if not isinstance(value, str):
+            raise ValueError("The value of lable must be string.")
+
+
+class In:
+    def __init__(self, values: List[str]):
+        validate_values(values)
+        self.values = values
+
+
+class NotIn:
+    def __init__(self, values: List[str]):
+        validate_values(values)
+        self.values = values
+
+
+class Exists:
+    def __init__(self):
+        pass
+
+
+class DoesNotExist:
+    def __init__(self):
+        pass
+
+
+class LabelMatchExpression:
+    """An expression used to select node by node's labels
+    Attributes:
+        key: the key of label
+        operator: In、NotIn、Exists、DoesNotExist
+    """
+
+    def __init__(self, key: str, operator: Union[In, NotIn, Exists, DoesNotExist]):
+        self.key = key
+        self.operator = operator
+
+
+LabelMatchExpressionsT = Dict[str, Union[In, NotIn, Exists, DoesNotExist]]
+
+
+@PublicAPI(stability="alpha")
+class NodeLabelSchedulingStrategy:
+    """The schedling strategy of node Affinity with labels
+    Simplified usage:
+    scheduling_strategy=NodeLabelSchedulingStrategy({
+          "region": IN("us"),
+          "gpu_type": EXISTS()
+    })
+    """
+
+    def __init__(
+        self, hard: LabelMatchExpressionsT, *, soft: LabelMatchExpressionsT = None
+    ):
+        self.hard = transfer_map_to_expresions(hard)
+        self.soft = transfer_map_to_expresions(soft)
+
+
+def transfer_map_to_expresions(map_expressions: LabelMatchExpressionsT):
```

Comment:
_convert_map_to_label_math_expressions

Reply:
I change to `_convert_map_to_expressions`.   more concise.